tests/e2e/commands/subnet.go

```diff
@@ -38,7 +38,16 @@ func CreateSubnetEvmConfigWithVersion(subnetName string, genesisPath string, ver
 	gomega.Expect(exists).Should(gomega.BeFalse())
 
 	// Create config
-	cmdArgs := []string{SubnetCmd, "create", "--genesis", genesisPath, "--evm", subnetName, "--" + constants.SkipUpdateFlag}
+	cmdArgs := []string{
+		SubnetCmd,
+		"create",
+		"--genesis",
+		genesisPath,
+		"--evm",
+		subnetName,
+		"--" + constants.SkipUpdateFlag,
+		"--teleporter=false",
```

Comment:
make teleporter=true|false param to pass to it can be used in e2e

Reply:
will do for teleporter e2e test